packages/govuk-frontend-review/src/views/layouts/layout.njk

```diff
@@ -14,7 +14,14 @@
     <script src="/vendor/iframe-resizer/iframeResizer.min.js" defer></script>
```

Comment:
Hmm, maybe we should defer this on the Design System site as well. We don't currently.

Reply:
Ah we added this when ES modules were introduced, since they have `defer` behaviour by default

Otherwise we saw regular `<script>` run ahead of GOV.UK Frontend `<script type="module">` code

I've pushed up an ES modules version instead 👍